package.json

```diff
@@ -7,8 +7,8 @@
     "@types/node": "^10.12.18",
     "@types/react": "^16.7.18",
     "@types/react-dom": "^16.0.11",
-    "react": "^16.7.0",
-    "react-dom": "^16.7.0",
+    "react": "^16.7.0-alpha.2",
```

Comment:
Using an alpha version might be a bit spooky. I totally get it if you don't wanna do this - which is why I provided a non-hooks example in the description.